docs/go-sdk-video-tutorial.md

```diff
@@ -138,5 +138,5 @@ func main() {
 Commands:
 
 ```bash
-docker run --network=host --rm temporalio/tctl:0.26.0 wf start --tl tutorial_tl -w Greet_Temporal_1 --wt Greetings --et 3600 --dt 10
+docker run --network=host --rm temporalio/tctl:0.25.0 wf start --tl tutorial_tl -w Greet_Temporal_1 --wt Greetings --et 3600 --dt 10
```

Comment:
```suggestion
docker run --network=host --rm temporalio/tctl:latest wf start --tl tutorial_tl -w Greet_Temporal_1 --wt Greetings --et 3600 --dt 10
```

Reply:
this would require to also change the "Installing Temporal" doc to install the latest. Which may be not a good idea since the latest doesn't guarantee full stability